src/components/RadioList.tsx

```diff
@@ -25,7 +25,7 @@ interface RadioListItemHintProps {
   value: string;
   children: ReactNode;
   hintText: string;
-  inputHtmlAttributes?: Record<string, string>;
+  inputHtmlAttributes?: Record<string, string | boolean>;
```

Comment:
Needed for radio buttons where `checked: false` and `checked: "false"` do not mean the same thing (the string "false" evaluates truthy).